### PR TITLE
Fixed species name input placeholder text not being translated

### LINKED ISSUE #1876: Blank species name textbox shows placeholder string
It shouldn't show the placeholder name here:

Instead the translated text should be displayed. Not sure if this is a godot problem (in which case a manual translation server call is needed) or there is a slight slipup somewhere in the text box configuration.

### PR DESCRIPTION
**Pull request by hhyyrylainen**
fixes #1876

### CODE REVIEW

File: locale/bg.po [hunk 1]
```diff
@@ -1991,6 +1991,11 @@ msgstr "Външност"
 msgid "BEHAVIOR"
 msgstr "Поведение"
 
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1568
+#, fuzzy
+msgid "SEARCH_DOT_DOT_DOT"
+msgstr "..."
```
Comment (Kasterisk) [inline locale/bg.po:1997]: The text for this used to be `Search...`

File: locale/bg.po [hunk 2]
```diff
@@ -2039,6 +2044,11 @@ msgstr "Мека / Твърда"
 msgid "COLOUR"
 msgstr "Цвят"
 
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:3925
+#, fuzzy
+msgid "SPECIES_NAME_DOT_DOT_DOT"
+msgstr "..."
```
Comment (Kasterisk) [inline locale/bg.po:2050]: And this was `Species Name...`

**Review by icedjuro — APPROVED** [review]
Works as intended

**Review by Kasterisk — DISMISSED** [review]

File: locale/bg.po (hunk 1 above)
Comment (icedjuro) [reply locale/bg.po:1997]: That's for Bulgarian? He put the correct translation for english

Comment (hhyyrylainen) [reply locale/bg.po:1997]: This is the bg translation, that was automatically generated by gettext (note the `#, fuzzy` part, which is added by gettext when it tries to guess a potential translation).

Comment (Kasterisk) [reply locale/bg.po:1997]: Oh huh, didn't see that woops, alright this should be good then.

**Review by Kasterisk — APPROVED** [review]

**Pull request merged**